localisation/00_zuwc_united_localisation.csv

```diff
@@ -2926,17 +2926,18 @@ RULE_SCOURGE_SPAWN_RANDOM;Random;;Zuf
 RULE_SCOURGE_SPAWN_RANDOM_DESC;The Scourge may not appear in some games since Ner'zhul dies on Draenor.;;Die Gei�el taucht m�glicherweise in einigen Spielen nicht auf, weil Ner'zhul auf Draenor stirbt.;;;;;;;;;;;x
 RULE_SCOURGE_SPAWN_ALWAYS;Always;;Immer;;;;;;;;;;;x
 RULE_SCOURGE_SPAWN_ALWAYS_DESC;The Scourge always appear on Azeroth after the destruction of Draenor (if the Scourge is enabled).;;Die Gei�el erscheint immer auf Azeroth nach der Zerst�rung von Draenor (wenn die Gei�el aktiviert ist).;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+form_the_gorian_empire;Form the Gorian Empire;;Gr�ndet das Gorianische Imperium;;;;;;;;;;;x
+form_the_gorian_empire_desc;Once, the Gorian Empire was the greatest state on Draenor. It is time to bring its legacy and glory to the brave new world.;;Einst war das gorianische Imperium die gr��te Nation auf Draenor. Es ist an der Zeit, sein Verm�chtnis und seinen Ruhm in die sch�ne neue Welt zu bringen.;;;;;;;;;;;x
```

Comment:
```suggestion
form_the_gorian_empire_desc;Once, the Gorian Empire was the greatest state on Draenor. It is time to bring its legacy and glory to this brave new world.;;Einst war das gorianische Imperium die gr��te Nation auf Draenor. Es ist an der Zeit, sein Verm�chtnis und seinen Ruhm in die sch�ne neue Welt zu bringen.;;;;;;;;;;;x
```
literally changed the to this before the word brave.

Reply:
done